src/components/Editor/tests/DebugLine.spec.js

```diff
@@ -15,7 +15,7 @@ function createMockDocument(clear) {
   const doc = {
     addLineClass: jest.fn(),
     removeLineClass: jest.fn(),
-    markText: jest.genMockFunction().mockReturnValue({ clear }),
+    markText: jest.fn(() => ({ clear })),
```

Comment:
`genMockFunction` was removed and was an undocumented API. Removing it from our examples and code.

https://github.com/facebook/jest/pull/6173